public/app/features/admin/UserPermissions.tsx

```diff
@@ -99,4 +97,7 @@ const getTooltipStyles = (theme: GrafanaTheme2) => ({
     font-style: italic;
     margin-right: ${theme.spacing(0.6)};
   `,
+  container: css`
+    margin-bottom: ${theme.spacing(5)};
+  `,
```

Comment:
```suggestion
  container: css({
    marginBottom: theme.spacing(5)
  }),
```
Using objects for the `css` functions enables better typing, so we have made it a convention now :)